webapp/src/components/contentBlock.tsx

```diff
@@ -27,29 +28,35 @@ import './contentBlock.scss'
 type Props = {
     block: IContentBlock
     card: Card
-    contents: readonly IContentBlock[]
     readonly: boolean
-    onDrop: (srctBlock: IContentBlock, dstBlock: IContentBlock) => void
+    onDrop: (srctBlock: IContentBlockWithCords, dstBlock: IContentBlockWithCords, position: Position) => void
+    width?: number
+    cords: {x: number, y?: number, z?: number}
 }
 
 const ContentBlock = React.memo((props: Props): JSX.Element => {
-    const {card, contents, block, readonly} = props
+    const {card, block, readonly, cords} = props
     const intl = useIntl()
-    const [isDragging, isOver, gripRef, itemRef] = useSortableWithGrip('content', block, true, props.onDrop)
+    const [, , gripRef, itemRef] = useSortableWithGrip('content', {block, cords}, true, () => {})
+    const [, isOver2,, itemRef2] = useSortableWithGrip('content', {block, cords}, true, (src, dst) => props.onDrop(src, dst, 'right'))
```

Comment:
why are we using the `useSortableWithGrip` instead of `useSortable` if we are not using the Grip?

Reply:
Talked this on mattermost, this is right.